src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1830,6 +1830,189 @@ <h2 class="text--center">Balance sheet</h2>
                 </tr>
 
               {{end}}
+              {{end}}
+
+              <!-- BALANCE SHEET - LIABILITIES -->
+
+              {{if $liabilities := $balanceSheet.other_liabilities_and_assets}}
+
+                  {{if $prepayments := $liabilities.prepayments_and_accrued_income}}
+                      <tr class="figures">
+                          <td id="prepayments-label">Prepayments and accrued income:</td>
```

Comment:
Same here. 
Do we need to match our ids to the ids in the provided SmallFullAccountsModifiediXBRL.html?